requirements/integration.cloud.vcenter.txt

```diff
@@ -1 +1,2 @@
 pyvmomi
+git+https://github.com/vmware/vsphere-automation-sdk-python.git
```

Comment:
These changes should be made in ansible/ansible first, and then updated by running `make update && make freeze` instead of manually editing them here.

Reply:
ok, good to know.